0-bootstrap/README.md

```diff
@@ -1,59 +1,151 @@
     1. Run `terraform plan -input=false -out bootstrap.tfplan`
     1. Run `terraform show -json bootstrap.tfplan > bootstrap.json`
     1. Run `terraform-validator validate bootstrap.json --policy-path="../policy-library" --project <A-VALID-PROJECT-ID>` and check for violations (`<A-VALID-PROJECT-ID>` must be an existing project you have access to, this is necessary because Terraform-validator needs to link resources to a valid Google Cloud Platform project).
-1. Run `terraform apply`
-1. Run `terraform output gcs_bucket_tfstate` to get your GCS bucket from the apply step
-1. Copy the backend by running `cp backend.tf.example backend.tf` and update `backend.tf` with your GCS bucket.
-1. Re-run `terraform init` agree to copy state to GCS when prompted
-    1. (Optional) Run `terraform apply` to verify state is configured correctly
+1. Run `terraform apply`.
+1. Run `terraform output gcs_bucket_tfstate` to get your Google Cloud bucket
+   from the previous step.
+1. Copy the backend:
 
-### (Optional) State backends for running terraform locally
+   ```
+   cp backend.tf.example backend.tf
+   ```
 
-Currently, the bucket information is replaced in the state backends as a part of the build process when executed by Cloud Build. If you would like to execute terraform locally, you will need to add your GCS bucket to the `backend.tf` files. You can update all of these files with the following steps:
+1. Update `backend.tf` with the name of your Cloud Storage bucket.
+1. Run `terraform output terraform_sa_email` to get the email address of the
+   admin. You need this address in a later procedure.
+1. Re-run `terraform init`. When you're prompted, agree to copy state to
+   Cloud Storage.
+1. (Optional) Run `terraform apply` to verify that state is configured
+   correctly. You should see no changes from the previous state.
 
-1. Change into the main directory for the terraform-example-foundation.
-1. Run this command ```for i in `find -name 'backend.tf'`; do sed -i 's/UPDATE_ME/GCS_BUCKET_NAME/' $i; done``` where `GCS_BUCKET_NAME` is the name of your bucket from the steps executed above.
+## Running Terraform locally
+
+If you deploy using Cloud Build, the bucket information is replaced in the state
+backends as a part of the build process when the build is executed by Cloud
+Build. If you want to execute Terraform locally, you need to add your Cloud
+Storage bucket to the `backend.tf` files. You can update all of these files with
+the following steps:
+
+1. Go to the `terraform-example-foundation` directory.
+1. Run the following command:
+
+   ```
+   for i in `find -name 'backend.tf'`; do sed -i 's/GCS_BUCKET_NAME/UPDATE_ME/' $i; done
```

Comment:
```suggestion
   for i in `find -name 'backend.tf'`; do sed -i 's/UPDATE_ME/GCS_BUCKET_NAME/' $i; done
```

@bharathkkb My bad, I was testing the command and pasted the one I used to revert the change in the suggestion, sorry.